PowerShellTools/Intellisense/Utilities.cs

```diff
@@ -147,25 +147,25 @@ internal static int GetCurrentBufferPosition(ITextView textView, out ITextBuffer
 
         internal static bool IsInCommentArea(int position, ITextBuffer buffer)
         {
-            return IsInCertainPSTokenTypesArea(position, buffer, PSTokenType.Comment);
+            return IsInCertainPSTokenTypesArea(position, buffer, false, PSTokenType.Comment);
         }
 
         internal static bool IsInStringArea(int position, ITextBuffer buffer)
         {
-            return IsInCertainPSTokenTypesArea(position, buffer, PSTokenType.String);
+            return IsInCertainPSTokenTypesArea(position, buffer, false, PSTokenType.String);
         }
 
         internal static bool IsInParameterArea(int position, ITextBuffer buffer)
         {
-            return IsInCertainPSTokenTypesArea(position, buffer, PSTokenType.CommandParameter);
+            return IsInCertainPSTokenTypesArea(position, buffer, true, PSTokenType.CommandParameter);
         }
 
         internal static bool IsInVariableArea(int position, ITextBuffer buffer)
         {
-            return IsInCertainPSTokenTypesArea(position, buffer, PSTokenType.Variable, PSTokenType.Member);
+            return IsInCertainPSTokenTypesArea(position, buffer, true, PSTokenType.Variable, PSTokenType.Member);
         }
 
-        private static bool IsInCertainPSTokenTypesArea(int position, ITextBuffer buffer, params PSTokenType[] selectedPSTokenTypes)
+        private static bool IsInCertainPSTokenTypesArea(int position, ITextBuffer buffer, bool edgeIncluded, params PSTokenType[] selectedPSTokenTypes )

```

Comment:
Ah, can scratch, didn't catch the fact that the last parameter set was a 'params'